src/App.js

```diff
@@ -4,7 +4,12 @@ import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import InvitationForm from './Invitation/InvitationForm';
 import Testing from './Invitation/Testing';
 import RSVP from './Invitation/RSVP';
+<<<<<<< HEAD
 import Registry from './registry/registry';
+=======
```

Comment:
Same as HEAD comment

Reply:
forgot to save after rebasing. should be fixed now